### PR TITLE
Post authentication hook

### PR DESCRIPTION
**Pull request by ankitrox**
Add the hook once the user has been authenticated successfully.

### CODE REVIEW

**Comment by renzit** [comment]
Would be nice to have! is there any workaround to redirect users to other than wp-admin?

**Comment by abhishekfdd** [comment]
Verified. 
Closes issue #89

File: src/Modules/Login.php [hunk 1]
```diff
@@ -143,6 +144,15 @@ public function authenticate( $user = null ) {
 			if ( $user instanceof WP_User ) {
 				$this->authenticated = true;
 
+				/**
+				 * Fires once the user has been authenticated via Google OAuth.
+				 *
+				 * @param WP_User $user WP User object.
+				 *
+				 * @since 1.2.2
```
Comment (chandrapatel) [inline src/Modules/Login.php:152]: This is the current version and this new action going to release in newer version. So maybe it's `1.2.3` or `1.3.0`?

**Review by chandrapatel — CHANGES_REQUESTED** [review]

File: src/Modules/Login.php (hunk 1 above)
Comment (chandrapatel) [reply src/Modules/Login.php:152]: @abhishekfdd

File: src/Modules/Login.php [hunk 2]
```diff
@@ -143,6 +144,15 @@ public function authenticate( $user = null ) {
 			if ( $user instanceof WP_User ) {
 				$this->authenticated = true;
 
+				/**
+				 * Fires once the user has been authenticated via Google OAuth.
+				 *
+				 * @param WP_User $user WP User object.
+				 *
+				 * @since 1.2.3
```
Comment (chandrapatel) [inline src/Modules/Login.php:152]: Ideally `@since` should be above `@param`. Also add blank line after `@since`. I missed this in last review. Apologies.

**Review by chandrapatel — CHANGES_REQUESTED** [review]

**Review by chandrapatel — APPROVED** [review]

**Pull request merged**